README.md

```diff
@@ -8,6 +8,23 @@ Epiphany can run on as few as one node (laptop, desktop, server) but the real va
 
 We currently use Terraform and Ansible for our automation orchestration. All automation is idempotent so you can run it as many times as you wish and it will maintain the same state unless you change the data. If someone makes a "snow flake" change to the environment (you should never do this) then simply running the automation again will put the environment back to the desired state.
 
+## Easy get started
+
+Fork `epiphany` repository and modify one of yaml's under `core/data/` directory. For example `data/azure/infrastructure/epiphany-playground/basic-data.yaml` you will need to modify few values in this file (like you azure subscription name, directory path for ssh keys). Once you done with `basic-data.yaml` you can execute Epiphany with command:
+
+```shell
+./epiphany -a -b -i -f infrastructure/epiphany-playground -t infrastructure/epiphany-template
+```
+
+This setup works on simplified file that is fine to start with, if you need more control of infrastructure created you should look to `data/azure/infrastructure/epiphany-bld-apps/data.yaml`.
```

Comment:
**a** simplified file  ..... control **over the** infra..... should look at.....